cmd/shell.go

```diff
@@ -197,11 +199,45 @@ func NewLogger(logConf SyslogConfig) blog.Logger {
 	return logger
 }
 
+func newVersionCollector() prometheus.Collector {
+	var buildTime string
+	if core.BuildTime == core.Unspecified {
+		// Will happen if Boulder wasn't built using the Makefile.
+		buildTime = core.Unspecified
+	} else {
```

Comment:
This could optionally be simplified to
```suggestion
	buildTime := core.Unspecified
	if core.BuildTime != core.Unspecified {
```